mysql-test/suite/innodb/t/alter_inplace_after_discard.test

```diff
@@ -0,0 +1,7 @@
+--source include/have_innodb.inc
+CREATE TABLE t (a INT) ENGINE=INNODB;
+ALTER TABLE t DISCARD TABLESPACE;
+ALTER TABLE t ENGINE INNODB;
+ALTER TABLE t ENGINE INNODB;
+RENAME table t to t1;
+DROP TABLE t1;
```

Comment:
The test could be named `alter_discard`. Please do not omit the newline character at the end of the last line.